docs/general/installation/container.md

```diff
@@ -311,28 +311,29 @@ SuccessExitStatus=0 143
 WantedBy=default.target
 ```
 
-## TrueNAS SCALE / TrueCharts
+## TrueNAS SCALE
 
-Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App inside the [TrueCharts](https://www.truecharts.org/) App Catalog with direct integration into the GUI, no CLI needed. Direct support is available on the [TrueCharts Discord](https://discord.gg/tVsPTHWTtr) and the source code is available on [GitHub](https://github.com/truecharts/charts).
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App in the official app catalog with direct integration into the GUI, no CLI needed.
 
-1. Install the TrueCharts Catalog to TrueNAS SCALE, see [website](https://truecharts.org/manual/SCALE/guides/getting-started/#adding-truecharts) for more info.
+1. Go to Apps page from the top level SCALE menu
 
-   1. Go to Apps page from the top level SCALE menu
-   2. Select Manage Catalogs tab on the Apps page
-   3. Click Add Catalog
-   4. After reading the iXsystems notice, click Continue and enter the required information:
-      Name: truecharts
-      Repository: `https://github.com/truecharts/catalog`
-      Preferred Trains: `enterprise` and `stable`
-      Branch: main
-   5. Click Save and allow SCALE to refresh its catalog with TrueCharts (this may take a few minutes)
+2. Click `Discover Apps` and search for `Jellyfin`
 
-2. Click `Available Applications` and search for `Jellyfin`
+3. Click `Install`, which will take you to the GUI installation wizard and you'll be able to fill out the necessary info
 
-3. Click `Install`, which will take you to the GUI Wizard and you'll be able to fill out the necessary info
+   - Published Server URL to publish in UDP Auto Discovery response
+   - User and Group Configuration, Network Configuration
+   - Storage Configuraton
+     - For `Jellyfin Config Storage` and `Jellyfin Cache Storage`, consider mapping Host Paths for ease of backup.
+       - Store them on SSD storage if possible, as using HDD storage **will** lead to poor experience.
+     - For `Jellyfin Transcode Storage`, consider using `Temporary` or `tmpfs`
+       - `Temporary` places a Docker volume under your configured Apps dataset (`Apps` -> `Configuration` -> `Choose Pool`). Avoid if that's on HDD.
+       - `tmpfs` creates a temporary directory on the RAM (but swap usage is permitted). Ensure you set the limit to at least a few gigabytes.
```

Comment:
```suggestion
       - `tmpfs` creates a temporary directory on the RAM (but swap usage is permitted). Ensure you set the limit to at least a few gigabytes. You may need to increase the limit if you have many large media files, or transcodes might fail.
```

Reply:
Okay I'll add that line. But now this makes me again wonder about the total limits, because I've read some forum posts online etc. and it might be that Docker app's total memory usage limit also applies to app+all tmpfs's combined! There isn't too much info about it online other than someone complaining that the behavior is not documented. If that's true then users might need to adjust the below too (but I'm really not sure if yes or no...):

![obraz](https://github.com/user-attachments/assets/79589d61-64f6-4201-ba4c-8a459827a96c)

Maybe I should add a new item there to say that if someone's boot disk is the only SSD in the system, to mount it to Host Path at `/var/tmp/jellyfin_transcodes`?

Reply:
You should maybe ask on the Truenas scale forums for clarification. If the memory limit excludes tmpfs then 4GB should be sufficient for most setups.

Reply:
I think I'll add "Note that total memory usage may also be capped in the `Resources Configuration` section." for the time being, so that hopefully people notice that if they were to still run into out of memory problems due to that.

We could ask on the forums, but I didn't find anything mentioned in the docs and iirc I glanced over how they configure Docker and they literally just put in these values to respective configuration sections there, which means it should behave like vanilla Docker, so with that I bet it will exhibit this behavior unless they were to end up adding size of user tmpfs's to the configured total memory limit...